src/api/transactionSubmission/build.ts

```diff
@@ -3,7 +3,11 @@
 
 import { AccountAddressInput } from "../../core";
 import { generateTransaction } from "../../internal/transactionSubmission";
-import { InputGenerateTransactionPayloadData, InputGenerateTransactionOptions } from "../../transactions";
+import {
+  InputGenerateTransactionPayloadData,
+  InputGenerateTransactionOptions,
+  InputGenerateOrderlessTransactionOptions,
```

Comment:
Does not seem like it is being used
```suggestion
  InputGenerateOrderlessTransactionOptions,
```

Reply:
Oh hm... I thought that was removed, some weird merge issue.  will fix